### DISCUSSION TITLE
Duplicate runs while using the PAT token

### DISCUSSION THREAD

**Discussion by joelmathewphilip** [stefanzweifel/git-auto-commit-action#304] [Help] [answered]
I have setup a trigger on a workflow which listens to changes on the main branch. The update is done using PAT token. But it results in a chain of loop runs one after the other, when changes are merged to the main branch. Can someone help me with this, without removing the trigger on the main branch. 

![image](https://github.com/stefanzweifel/git-auto-commit-action/assets/45589731/2bc5174b-c1ef-491f-9535-fe9aea576b1d)

**Comment by stefanzweifel** [comment, marked as answer]
You either have to add `[skip ci]` to the commit message or add an `if`-clause to your workflow file.

See this discussion: https://github.com/stefanzweifel/git-auto-commit-action/discussions/298#discussioncomment-6680182
